### PR TITLE
[feat] Add a slider to show multiple objects in a single box

### PR DESCRIPTION
**Pull request by KaroMourad**
- Add a depth slider component to be able to slide and view objects, grouped by the same position
- Add the same slider to full-view mode as well

Improve the global hooks folder by adding an `index.ts` file and export all hooks from it

### CODE REVIEW

**Comment by KaroMourad** [comment]
Resolves #1731

File: aim/web/ui/src/components/DepthSlider/DepthSlider.tsx [hunk 1]
```diff
@@ -1,42 +1,51 @@
...
+      if (typeof onDepthChange === 'function') {
+        onDepthChange(value, index);
+      }
+    },
+    [onDepthChange, index],
+  );
+  return items.length === 0 ? null : (
     <ErrorBoundary>
-      <div className='DepthSlider' style={{ height: MEDIA_SET_SLIDER_HEIGHT }}>
+      <div className={`DepthSlider ${className}`} style={style}>
```
Comment (VkoHov) [inline aim/web/ui/src/components/DepthSlider/DepthSlider.tsx:33]: maybe we can use classNames here?

File: aim/web/ui/src/components/DepthSlider/DepthSlider.tsx [hunk 2]
```diff
@@ -1,42 +1,51 @@
...
+          valueLabelDisplay={valueLabelDisplay}
+          getAriaValueText={(value) => `${items[value]}`}
           value={depth}
-          onChange={(e, value) => onDepthChange?.(value as number, index)}
+          onChange={(e, value) => onChange(value as number)}
           step={null}
           marks={sliderMarks}
           min={0}
-          max={pathValue.length - 1}
+          max={items.length - 1}
```
Comment (VkoHov) [inline aim/web/ui/src/components/DepthSlider/DepthSlider.tsx:45]: Duplicated max value "items.length - 1" on this line and on line 61

File: aim/web/ui/src/modules/BaseExplorer/components/BoxWrapper/BoxWrapper.scss [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  position: absolute;
+  box-shadow: 0 0 0 0.0625rem $primary-color;
+  &__box {
+    height: 100%;
+    width: 100%;
+    overflow: auto;
+  }
+  &__fullScreen {
+    position: absolute;
+    top: 0.25rem;
```
Comment (VkoHov) [inline aim/web/ui/src/modules/BaseExplorer/components/BoxWrapper/BoxWrapper.scss:13]: Let's change it to $space-xxxs variable

File: aim/web/ui/src/modules/BaseExplorer/components/Visualizer/Visualizer.tsx [hunk 4]
```diff
@@ -1,11 +1,15 @@
 import * as React from 'react';
-import _ from 'lodash-es';
+import * as _ from 'lodash-es';
```
Comment (VkoHov) [inline aim/web/ui/src/modules/BaseExplorer/components/Visualizer/Visualizer.tsx:2]: Why we can't import it as 'import _ from 'lodash-es';'

**Review by rubenaprikyan — APPROVED** [review]
Great work!

**Pull request merged**